frontend/src/__tests__/cypress/cypress/tests/mocked/connectionTypes/createConnectionType.cy.ts

```diff
@@ -29,15 +32,39 @@ describe('create', () => {
     createConnectionTypePage.findConnectionTypePreviewToggle().should('exist');
   });
 
-  it('Allows create button with valid name', () => {
+  it('Allows create button with valid name and category', () => {
+    const categorySection = createConnectionTypePage.getCategorySection();
     createConnectionTypePage.visitCreatePage();
 
     createConnectionTypePage.findConnectionTypeName().should('have.value', '');
     createConnectionTypePage.findSubmitButton().should('be.disabled');
 
     createConnectionTypePage.findConnectionTypeName().type('hello');
+    categorySection.findCategoryTable().click();
+    connectionTypesPage.findMultiGroupSelectButton('Object-storage');
     createConnectionTypePage.findSubmitButton().should('be.enabled');
   });
+
+  it('Selects category or creates new category', () => {
+    createConnectionTypePage.visitCreatePage();
+
+    const categorySection = createConnectionTypePage.getCategorySection();
+
+    categorySection.findCategoryTable().click();
+    connectionTypesPage.findMultiGroupSelectButton('Object-storage');
+
+    categorySection.findChipItem(/^Object storage$/).should('exist');
+    categorySection.clearMultiChipItem();
+
+    connectionTypesPage.findMultiGroupSelectButton('Object-storage');
+
+    categorySection.findMultiGroupInput().type('Database');
+    connectionTypesPage.findMultiGroupSelectButton('Database');
+
+    categorySection.findMultiGroupInput().type('New category');
```

Comment:
After typing in a new category, you should find the creation options to create the new entry.